recite/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class ReciteConfig(AppConfig):
```

Comment:
I have never seen anyone calling his class with a verb :)

Reply:
We have Reciters and Recitations models in this class. Didn't found noun to include both.

Reply:
So you took the common prefix xD
`RecitationConfig` sounds good :)